pulp-template.yaml

```diff
@@ -164,4 +166,10 @@ objects:
 parameters:
   - name: ENV_NAME
     description: Specify your (ephemeral) namespace
-    eequired: true
+    required: true
+  - name: IMAGE_NAME
+    description: Specify which container image the operator will deploy as Pulp services.
+    required: true
+  - name: IMAGE_VERSION
+    description: Specify the tag or hash for the image deployed by the operator.
+    required: true
```

Comment:
I think we should specify the default value instead of making it required.

Reply:
This could default to "latest"

Reply:
Yup